react/features/base/ui/constants.any.ts

```diff
@@ -13,6 +13,8 @@ export enum BUTTON_TYPES {
  */
 export const BUTTON_MODES: {
     CONTAINED: 'contained';
+    TEXT: 'text';
 } = {
-    CONTAINED: 'contained'
+    CONTAINED: 'contained',
```

Comment:
ISss there no better way to annotate this?

Reply:
Can you be more specific, please? I don't get it.

Reply:
The type description is bacially the same as the content. I wonder if there is no other way around it?

Reply:
We use react-native-paper for buttons and these are the modes that Button component accepts.
https://callstack.github.io/react-native-paper/button.html